components/automate-backend-deployment/habitat/plan.sh

```diff
@@ -16,14 +16,14 @@ pkg_deps=(
   core/gawk
   core/gzip
   core/jq-static
-  core/openssh
-  core/openssl
+  core/openssh/7.9p1/20220311132702
+  core/openssl/1.0.2zb/20220311111046
```

Comment:
in the latest dev, we have upgraded to openssl 1.0.2zf version, please test with that version. 
Never pin a version for packages

Reply:
Hi @Dmaddu, the changes were made to test the terraform version after making changes in the code. I will revert it back to chef.

Reply:
Removed the pinned version and reverted the origin of the package to its original.